vvvv45/addonpack/src/nodes/plugins/Animation/DelayFramebased/DelayFramebasedNode.cs

```diff
@@ -0,0 +1,117 @@
+#region usings
+using System;
+using System.ComponentModel.Composition;
+
+using System.Collections.Generic;
+using System.Collections;
+
+using VVVV.PluginInterfaces.V1;
+using VVVV.PluginInterfaces.V2;
+using VVVV.Utils.VColor;
+using VVVV.Utils.VMath;
+
+using VVVV.Core.Logging;
+#endregion usings
+
+namespace VVVV.Nodes
+{
+	#region PluginInfo
+	[PluginInfo(Name = "Delay", Category = "Animation", Version ="Framebased", Help = "Framebased delay with adjustable frame count")]
+	#endregion PluginInfo
+	public class DelayFramebased : IPluginEvaluate
+	{
+		#region fields & pins
+		[Input("Input", DefaultValue = 1.0)]
+		public ISpread<double> FInput;
+		
+		[Input("Skip")]
+		public ISpread<bool> FSkip;
+		
+		[Input("Reset")]
+		public ISpread<bool> FReset;
+
+		[Input("Delay", DefaultValue = 1.0, MinValue = 0)]
```

Comment:
in the Delay (Animation) node this parameter is called "Time". for consistency reasons i'd suggest to name this pin "Frames"